joomla/joomla-cms
# [4.3][Guided Tours] Cleanup Tours model

Pull Request for Issue # .

### Summary of Changes
This PR makes several clean up to Tours module from Guided Tours extension 

- No need to override `getEmptyStateQuery` method from parent class (This override contains really wrong code, seems copied from Steps model)
- Remove getTable method. We do not need this method from a List model

### Testing Instructions
- Install Joomla from download package generated by this PR (to make sure Guided Tours is available on your installation). Access to Guided Tours, see list of tours and make sure nothing is broken from this page 

### Actual result BEFORE applying this Pull Request
Unnecessary wrong code in Tours model

### Expected result AFTER applying this Pull Request
The unnecessary wrong code removed.

### Link to documentations
Please select:
